annotations/annotations_rw_client_test.go

```diff
@@ -19,6 +20,7 @@ import (
 )
 
 var testingClient = fthttp.NewClientWithDefaultTimeout("PAC", "test-annotations-publisher")
+var draftsURL = "/drafts/content/:uuid/annotations"
```

Comment:
To minimize the possibility of test dependencies we should probably use `const` more often.
```suggestion
const draftsURL = "/drafts/content/:uuid/annotations"
```